ui/app/AppLayouts/Chat/views/ChatMessagesView.qml

```diff
@@ -237,12 +252,15 @@ Item {
 
         model: messageStore.messagesModel
 
-        // Not Refactored Yet
-        //Component.onCompleted: scrollToBottom(true)
+        Component.onCompleted: chatLogView.scrollToBottom(true)
 
         delegate: MessageView {
             id: msgDelegate
 
+            // We wait to load all messages, because switching back to chats makes
+            // the scroll go crazy so it loads way too many messages making it slow
+            visible:  index < 10 || root.allMessagesLoaded
```

Comment:
Not sure how limiting to 10 visible delegates helps. One issue that I see and might be the root cause is that we don't define a delegate height and ListView has no idea how many delegates to instantiate. Do we somehow define the `implicitHeight` or `heigh` for a delegate somewhere where I didn't find it?